CONTRIBUTING.md

```diff
@@ -36,7 +36,7 @@ There is always something more that is required, to make it easier to suit your
 
 ## Contributing to Source Code and Bug Fixes
 
-Provide PRs with appropriate tags for bug fixes or enhancements to the source code. For a list of tags that could be used, see [this]((./contribute/labels-of-issues.md)).
+Provide PRs with appropriate tags for bug fixes or enhancements to the source code. For a list of tags that could be used, see [this](./contribute/labels-of-issues.md).
 
```

Comment:
Also, I think this change is already introduced in #2058. Let's solve the `.gitignore` issue only in this PR?

Reply:
Yeah sure. It was added by mistake.
Closing this PR. Will open a PR with only the .gitignore part